### PR TITLE
qt 6.1.0

### PR DESCRIPTION
**Pull request by ericLemanissier**
Specify library name and version:  **qt/6.1.0**

fixes conan-io/conan-center-index#5518

---

- [X] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [X] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [X] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [X] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

File: recipes/qt/6.x.x/conanfile.py [hunk 1]
```diff
@@ -384,6 +397,7 @@ def _configure_cmake(self):
 
         self._cmake.definitions["INSTALL_MKSPECSDIR"] = os.path.join(self.package_folder, "res", "archdatadir", "mkspecs")
         self._cmake.definitions["INSTALL_ARCHDATADIR"] = os.path.join(self.package_folder, "res", "archdatadir")
+        self._cmake.definitions["INSTALL_LIBEXECDIR"] = os.path.join(self.package_folder, "res", "archdatadir", "bin" if self.settings.os == "Windows" else "libexec")
```
Comment (prince-chrismc) [inline recipes/qt/6.x.x/conanfile.py:400]: Whats the motivation behind this change?

**Review by prince-chrismc — APPROVED** [review]

File: recipes/qt/6.x.x/conanfile.py (hunk 1 above)
Comment (ericLemanissier) [reply recipes/qt/6.x.x/conanfile.py:400]: it mimics this https://github.com/qt/qtbase/blob/4b60cea6020c99dabb3b2777e30f0a3924dcd8a7/src/corelib/global/qlibraryinfo.cpp#L320-L324 and this https://github.com/qt/qtbase/blob/4b09522c23e9efdf83ba8d4af436d8a700ccb66e/cmake/QtBuild.cmake#L112-L116
This folder is the one storing moc and other executables in qt 6.1.0

Comment (prince-chrismc) [reply recipes/qt/6.x.x/conanfile.py:400]: Is it only for >=6.1.0?

Comment (ericLemanissier) [reply recipes/qt/6.x.x/conanfile.py:400]: well, with this PR, it is for all versions.

**Review by prince-chrismc — APPROVED** [review]

**Review by prince-chrismc — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Review by uilianries — APPROVED** [review]

**Pull request merged**